src/ansible_builder/cli.py

```diff
@@ -164,12 +187,11 @@ def add_container_options(parser):
 
         n.add_argument('-v', '--verbosity',
                        dest='verbosity',
-                       type=int,
-                       choices=[0, 1, 2, 3],
+                       action=CustomVerbosityAction,
+                       nargs='?',
```

Comment:
`nargs='?'` is  potentially a big hammer here, since it can falsely trip when it's the last non-positional arg followed by a positional arg (and I can't recall offhand if you can "push" a token back for the next subparser). `--` makes it work, but a lot of people don't understand that.

I *think* at least for builder, It's actually OK right now, since none of our existing command subparsers currently support trailing posargs, but just something to acknowledge (as well as perhaps a good reason to actually deprecate the numeric syntax?), and to make sure that an existing runner subparser doesn't have this problem.

Reply:
It should be cool for now b/c we don't have positional args, but I did add deprecating the old style in a new deprecation tracking issue https://github.com/ansible/ansible-builder/issues/565.